README.md

```diff
@@ -359,6 +359,51 @@ container.openPath('game.html');
 any unsupported values. See [the SpringRoll Application Class docs](https://github.com/SpringRoll/SpringRoll/tree/v2/src#handling-state-change)
 for more information on the request format.
 
+### Fullscreen Plugin
+The full screen plugin hooks up a button to set an element to full screen then communicate this through Bellhop. The plugin will also add the class ```'--fullScreen'``` to the button given
```

Comment:
Could `button` be any HTML element, or is it only allowed to be `<button>`?

Reply:
Good point, in the current implementation it can be either